jvcl/run/JvDBGrid.pas

```diff
@@ -19,7 +19,7 @@
   Polaris Software
   Lionel Reynaud
   Flemming Brandt Clausen
-  Fr�d�ric Leneuf-Magaud
+  Frédéric Leneuf-Magaud
```

Comment:
This line should not have been changed, it seems that you inadvertently changed the encoding of the file from what it was to something else.
Could you please restore the original encoding?